runner/runner.go

```diff
@@ -265,6 +266,14 @@ type Runner struct {
 	credentials map[string]config.Github
 }
 
+func (r *Runner) GetControllerID() (uuid.UUID, error) {
```

Comment:
This can be removed if you give the collector access to the store.

As a side note, all functions should accept ```context.Context```, for at least the following reasons:

  * The context may get canceled and if we pass it along, it allows the various libraries we use to cancel their tasks and return gracefully.
  * It allows us to do some authorization checks

Reply:
I didn't give the collector access directly to the store, for now this is not necessary, access to the runner seems to be a good level of abstraction.

Reply:
I agree. See above wall of text :smile: